css/index.css

```diff
@@ -104,18 +215,67 @@ p {
 .middle-content .boxes .box {
     width: 12.5%;
     height: 100px;
-    background: black;
+    /* background: black; */
     margin: 20px 2.5%;
     color: white;
     display: flex;
     align-items: center;
     justify-content: center;
 }
 
```

Comment:
Nice job only overriding what the child needs to change (the color) and inheriting the rest of the styling from the parent.